sys-auth/ssh-ldap-pubkey/ssh-ldap-pubkey-1.3.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+inherit distutils-r1
+
+DESCRIPTION="Utility to manage SSH public keys stored in LDAP"
+HOMEPAGE="https://github.com/jirutka/ssh-ldap-pubkey"
+
+if [[ ${PV} == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/jirutka/${PN}/${PN}.git"
+
+	PATCHES=()
+
+	inherit git-r3
+else
+	SRC_URI="https://github.com/jirutka/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="test"
+
+CDEPEND="
```

Comment:
`CDEPEND` is reserved for future EAPI use.

Reply:
Hu? We can reserve names for future EAPI?! Anyway, we have 3653 ebuilds using CDEPEND... I would say it is common practice and not doing any harm. Not?

Reply:
You never know what will happen in a future EAPI. But yeah, because it is global scope, I'll prefix with "MY_".